manifests/x/XiamenYajiSoftwareCoLtd/CocosDashboard/1.2.1.2819/XiamenYajiSoftwareCoLtd.CocosDashboard.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: XiamenYajiSoftwareCoLtd.CocosDashboard
+PackageVersion: 1.2.1.2819
+PackageLocale: en-US
+Publisher: Xiamen Yaji Software Co. Ltd
+PackageName: Cocos Dashboard
+License: BSD
+Copyright: Copyright (C) 2022 Xiamen Yaji Software Co., Ltd.
+ShortDescription: This installer database contains the logic and data required to install Cocos Dashboard.

```

Comment:
Please use the existing metadata from `XiamenYaji.Cocos.Dashboard` for the License, Description, Etc.